### DISCUSSION TITLE
Training and inferencing at a particular resolution

### DISCUSSION THREAD

**Discussion by Rajesh-ParaxialTech** [Project-MONAI/MONAI#8114] [Q&A]
Hello
Suppose I want to train a lung nodule detection model using 

https://github.com/Project-MONAI/model-zoo/releases/download/hosting_storage_v1/lung_nodule_ct_detection_v0.5.9.zip.   Say, i want to train at a resolution of 0.7 mm x 0.7 mm x 1.0 mm.   using 

https://github.com/Project-MONAI/tutorials/blob/main/detection/luna16_prepare_images.py

and

 https://github.com/Project-MONAI/tutorials/blob/main/detection/luna16_prepare_env_files.py.

Right ?

Does this imply :   
1.       I must resample my training dataset to this resolution and then involve it in training.
2.     while inferencing also i must resample any test CT scan image to this resolution (0.7 mm x 0.7 mm x 1.0 mm)  and only then subject the CT scan to inference, for best performance ?

Thank you
Rajesh

**Comment by KumoLiu** [comment]
Hi @Rajesh-ParaxialTech, you can use a custom resolution for training without any issues. However, please note that if you plan to use the pretrained weights from the bundle, the results may not meet your expectations due to the resolution mismatch.
Thanks.
